test/test_cli.py

```diff
@@ -2,6 +2,8 @@
 The main testing script. Runs the devnet and calls its endpoints.
```

Comment:
As already noted in pyproject.toml comment, change the name of this and similar files.